KabithaC/00 - Projects/04 - Frontend Projects/HospitalManagementSystem/Assets/Javascript/Authentication.Admin/AddDoctor.js

```diff
@@ -0,0 +1,53 @@
+const DOCTOR_NAME = document.getElementById("doctorUserName");
+const DOCTOR_GENDER_MALE = document.getElementById("genderMale");
+const DOCTOR_GENDER_FEMALE = document.getElementById("genderFemale");
+const DOCTOR_SPECIALIST = document.getElementById("doctorSpecialist");
+const DOCTOR_EDUCATION = document.getElementById("doctorEducation");
+const DOCTOR_EXPERIENCE = document.getElementById("doctorExperience");
+const DOCTOR_EMAIL = document.getElementById("doctorEmailId");
+const DOCTOR_MOBILE_NUMBER = document.getElementById("doctorMobileNumber");
+
+let AdminMethods = {};
+
+AdminMethods.AddDoctor = function()
+{
+    let lstAdminDoctorlist = JSON.parse(localStorage.getItem("admin_doctor_register")) || {};
+
+    if(!AdminMethods.ValidateDoctor(lstAdminDoctorlist))
+    {
+        alert("Doctor already exists.");
```

Comment:
AdminMethods.ValidateDoctor(lstAdminDoctorlist) - Just call this method. No need for an If condition

Reply:
Done...